.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
+.env.template
+.vscode
+.gitignore
+README.md  
```

Comment:
Nice. Some other bits and pieces we might want to ignore include any local venv folders, and/or vagrant folders

Reply:
Added - 809fb0e